Load/lib/perl/InstallSchemaUtils.pm

```diff
@@ -0,0 +1,183 @@
+package ApiCommonData::Load::InstallSchemaUtils;
+
+use Exporter;
+use DBI;
+
+our @ISA= qw( Exporter );
+our @EXPORT = qw( runSql dropSchemaSetTables dropSchemaSetPostgres );
+
+sub runSql {
+  my ($login, $password, $db, $dbHostname, $dbVendor, $file, $allowFailures, @params) = @_;
+  
+  if (lc $dbVendor eq 'oracle') { 
+    &runSqlOracle($login, $password, $db, $dbVendor, $file, $allowFailures, @params);
+  } elsif (lc $dbVendor eq 'postgres') {
+    &runSqlPostgres($login, $password, $db, $dbHostname, $dbVendor, $file, $allowFailures @params);
+  } else { 
+    die "Unsupported dbVendor:$dbVendor."; 
+  }
+}
+
+sub runSqlOracle {
+  my ($login, $password, $db, $dbVendor, $file, $allowFailures, @params) = @_;
+
+  my $fullFile = "$ENV{GUS_HOME}/lib/sql/apidbschema/$dbVendor/$file";
```

Comment:
Instead of duplicating code here & runSqlPostgres, we can create the $fullFile in runSql since we already know the $dbVendor.

Reply:
good idea